src/game/gui/component.h

```diff
@@ -12,6 +12,7 @@
 #include "controller/controller.h"
 #include "game/gui/theme.h"
 #include <SDL.h>
+#include <game/gui/text/text.h>
```

Comment:
Why this one in `<>` but others in quotes?

Reply:
yeah, it should be in quotes.
i think this got added automatically by VS.

Reply:
Good guess! It was added by CLion :P

Reply:
Fixed now